autoscheduler/scraper/views.py

```diff
@@ -1,4 +1,14 @@
-# Once we start adding routes, uncomment this line
-# from django.shortcuts import render
+from rest_framework import generics
+from .serializers import CourseSerializer
+from .models.course import Course
 
-# Create your views here.
+class RetrieveCourseView(generics.RetrieveAPIView):
+    """ API endpoint for viewing course information """

```

Comment:
```suggestion
    """ API endpoint for viewing course information 
         Used by /api/course
    """
```

Doesn't have to be formatted like this, but the view docstring should contain what route it's used.

Reply:
Also probably explain that this view, given a dept, course_num, and term, returns the respective Course's title & credit hours.